File: app/src/main/java/com/pdamkotasmg/happywork/fitur/splash/model/DataItem.java
package com.pdamkotasmg.happywork.fitur.splash.model;

import com.google.gson.annotations.SerializedName;

public class DataItem{

    @SerializedName("app_name")
    private String appName;

    @SerializedName("updated_at")
    private String updatedAt;

    @SerializedName("package_name")
    private String packageName;

    @SerializedName("created_at")
    private String createdAt;

    @SerializedName("id")
    private Integer id;

    @SerializedName("deleted_at")
    private Object deletedAt;

    public void setAppName(String appName){
        this.appName = appName;
    }

    public String getAppName(){
        return appName;
    }

    public void setUpdatedAt(String updatedAt){
        this.updatedAt = updatedAt;
    }

    public String getUpdatedAt(){
        return updatedAt;
    }

    public void setPackageName(String packageName){
        this.packageName = packageName;
    }

    public String getPackageName(){
        return packageName;
    }

    public void setCreatedAt(String createdAt){
        this.createdAt = createdAt;
    }

    public String getCreatedAt(){
        return createdAt;
    }

    public void setId(Integer id){
        this.id = id;
    }

    public Integer getId(){
        return id;
    }

    public void setDeletedAt(Object deletedAt){
        this.deletedAt = deletedAt;
    }

    public Object getDeletedAt(){
        return deletedAt;
    }
}

File: app/src/main/java/com/pdamkotasmg/happywork/fitur/splash/model/packageName/DataItem.kt
package com.pdamkotasmg.happywork.fitur.splash.model.packageName

import com.google.gson.annotations.SerializedName

data class DataItem(

    @field:SerializedName("app_name")
    val appName: String? = null,

    @field:SerializedName("updated_at")
    val updatedAt: String? = null,

    @field:SerializedName("user_id")
    val userId: String? = null,

    @field:SerializedName("package_name")
    val packageName: String? = null,

    @field:SerializedName("created_at")
    val createdAt: String? = null,

    @field:SerializedName("id")
    val id: Int? = null,

    @field:SerializedName("deleted_at")
    val deletedAt: String? = null
)